scripts/qc_report_per_sample.rmd

```diff
@@ -20,11 +20,18 @@ library(stringr)
 library(magrittr)
 library(openssl)
 ```
+
 # Input Settings
 ```{r InputSettings, echo = FALSE, include = FALSE}
 coverage_file <- params$coverage_file
 sample_name <- params$sample_name
 outFile <- params$outFile
+fastqc_dir <- params$fastqc_dir # TODO @Jan @Miri add that argument to the snakemake rule and the rendersite function?
+# TODO: @Jan snakemake needs to be run twice before and after the preprocessing
```

Comment:
What are the inputs for each of the runs?

Reply:
What do you mean?

Reply:
You wrote "snakemake needs to be run twice before and after the preprocessing".  How do these runs differ in terms of inputs and outputs?

Reply:
in the first run the inputs are the raw_data.fastq files. 

In the second run the inputs would be the prinseq-outputs, the files with the preprocessed reads (are they bam already?)

Reply:
As output in both cases I need the fastqc_report.html files